guide/popular-topics/common-questions.md

```diff
@@ -377,3 +377,52 @@ Assuming the process is to be done for the guild the message is sent in.
 ```
 
 </branch>
+
+### How do I check the ping of my bot?
+
+There are several ways to get the **"ping"** of the bot. Two of the ways can be by the **WebSocket ping** or by the **message latency**.
```

Comment:
websocket heartbeat, API latency (not only message latency, this applies to all full roundtrips)

Reply:
Made adjustments for all of the requested changes, pushing after resolving.